src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/IExceptionInstance.cs

```diff
@@ -14,5 +15,7 @@ internal interface IExceptionInstance
         string TypeName { get; }
 
         DateTime Timestamp { get; }
+
+        CallStackResult CallStackResult { get; }
```

Comment:
This should be the `CallStackResult` from `Microsoft.Diagnostics.Monitoring.WebApi.Models` not from `Microsoft.Diagnostics.Monitoring.WebApi.Stacks`. The `IExceptionInstance` needs to be reporting realized information instead of numeric identifiers e.g. the stack frame should have the function name already resolved instead of providing a `FunctionId` property.

Reply:
I can do this, but doesn't the /stacks endpoint only do this translation when it's being formatted? I reused the logic being used there by doing the translation in `FormatStack`. Or do we want this to behave differently than that?